src/validation/forms/zodUserActionFormEmailCongressperson.ts

```diff
@@ -13,7 +13,14 @@ const base = object({
     .min(1, 'Please enter a message')
     .max(2000, 'Your message should not exceed 2000 characters'),
   subject: string().trim(),
-  dtsiSlugs: array(zodDTSISlug).min(1),
+  dtsiSlugs: array(zodDTSISlug)
+    .min(1)
+    .describe(
+      JSON.stringify({
+        triggerException: true,
+        message: 'Something went wrong with the DTSI Slug. Please check against database.',
+      }),
+    ),
```

Comment:
I would abstract this describe logic into a wrapper, would prefer an api that looks something like this
```suggestion
  dtsiSlugs: withMetadata(
      array(zodDTSISlug).min(1),
      {
        triggerException: true,
        message: 'Something went wrong with the DTSI Slug. Please check against database.',
      },
    ),
```

Reply:
Idea being that we'd have another function called `parseMetadata` or `getSchemaMetadata` that would work as a parse and that way all this description and `JSON.stringify` logic would be encapsulated

Reply:
I tried doing something like that but since this is all chainable from zod's, I don't think we can do much here you know? any thoughts?